pkgs/tools/security/creddump/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, python27, python27Packages }:
+
+python27Packages.buildPythonApplication rec {
+  pname = "creddump";
+  version = "0.3";
```

```suggestion
  version = "0.3";
  
```

---

Fixed

---

If there are no releases at https://github.com/moyix/creddump/releases, then what is the meaning of this? Ususally, with unstable versions of package in Nixpkgs, we use:

```suggestion
  version = "unstable-${date of last commit}";
```

---

`0.3` is the latest official release of `creddump`, which is reflected in that git repo. You can see it in the commit comments + repo description. 
However, I agree it would have been nice if they had added a tag.

---

Well it seems there are some differences between the two - moyix's version had an (real) update on 2017, while kali's version had a real change made at 2012.. see https://github.com/moyix/creddump/commits/master and https://github.com/moyix/creddump/commits/master . I don't care much which you'll choose, but if you'll decide to rechange to the GitHub one - use `version = unstable-...`  as suggested.